src/pages/appointmentsTest/components/ClickedDateModal.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Modal } from "antd";
+import dayjs from "dayjs";
+
+interface ClickedDateModalProps {
```

Comment:
what do you think about change this name? maybe "OnClickDateModalProps" or "OnClickedDateModalProps"